test/e2e/flask/multichain-api/password-locked.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { strict as assert } from 'assert';
+import {
+  ACCOUNT_1,
+  WALLET_PASSWORD,
+  WINDOW_TITLES,
+  withFixtures,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+  escapeColon,
+  type FixtureCallbackArgs,
+  getExpectedSessionScope,
+  getSessionScopes,
+  openMultichainDappAndConnectWalletWithExternallyConnectable,
+  passwordLockMetamaskExtension,
+} from './testHelpers';
+
+// Skipping these until we add this prompt to unlock behavior to the EIP-1193 API as well
+describe.skip("A dapp is connected with account and chain permissions previously granted via `wallet_createSession`, user's extension becomes password locked", function () {
```

Comment:
If this doesn't exist yet, perhaps we can omit it from this PR?

Reply:
Done here https://github.com/MetaMask/metamask-extension/pull/27782/commits/4feca478cd42f9c85deeccfc05133043d9fc3a83

Reply:
Re-introduced